modules/nix/linux-builder.nix

```diff
@@ -11,6 +11,8 @@ let
     modules = [ cfg.config ];
   };
 
+  WorkingDirectory = "/var/lib/darwin-builder";
```

Comment:
I think it would be good if you could make this into an option

Reply:
done.